src/main/com/example4/radek/DrawMeSquareApplication.java

```diff
@@ -0,0 +1,90 @@
+package main.com.example4.radek;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class DrawMeSquareApplication {
```

Comment:
kdyz zadam cislo nefunguje to

Reply:
Nerozumím ? Nepodařilo se mi dostat do takového stavu

Reply:
uz to jde

Reply:
mas tu 8 commitu, kdyz vyzkousis kazdy zvlast, tak bys to asi v jednom objevil